node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/ContainerOperationsImpl.java

```diff
@@ -243,28 +243,28 @@ public CommandResult executeCommandInNetworkNamespace(NodeAgentContext context,
     }
 
     @Override
-    public void resumeNode(NodeAgentContext context) {
-        executeNodeCtlInContainer(context, "resume");
+    public String resumeNode(NodeAgentContext context) {
+        return executeNodeCtlInContainer(context, "resume").getOutput();
```

Comment:
Return `Optional<String>` from here and `executeNodeCtlInContainer()` so we dont need to check `isBlank()` everywhere?

Reply:
The only way to avoid testing for isBlank() everywhere would be to add a method that returns Optional<String> from e.g. ProcessResult.  I don't know -- the output of a program is best modelled as String, not Optional<String> IMO.  It's the invoker of the particular program that knows how to interpret the output, and in these cases and for now I'm thinking isBlank => ignore, but that may change once this rolls out.